app/driver/onewire.c

```diff
@@ -180,7 +180,6 @@ void onewire_write(uint8_t pin, uint8_t v, uint8_t power /* = 0 */) {
   if ( !power) {
```

Comment:
you are probably right. I did not look into the code but only removed the line that caused the bug.    Will you prefer I also fix this?